test/event_reporter_test.rb

```diff
@@ -16,35 +16,28 @@ def test_reporter_can_find_all_Johns
     assert_equal 63, result.count
   end
 
-  def test_queue_can_be_cleared
-    er = EventReporter.new('./data/full_event_attendees.csv')
-    list = List.new
-
-    er.find_attendees(:first_name, "Rachel")
-    refute er.list.empty?
-
-    list.clear_queue
-    assert_equal 0, list.queue_count
-  end
-
   def test_reporter_can_find_all_Marys_and_queue_them
     er = EventReporter.new('./data/full_event_attendees.csv')
-
     result = er.find_attendees(:first_name, "Mary")
 
     assert_equal 16, result.count
     refute er.list.empty?
   end
 
-  def test_queue_can_be_sorted
+  def test_queue_can_be_sorted_and_first_in_sort_has_20009_zipcode
     er = EventReporter.new('./data/attendees_fixture.csv')
     er.find_attendees(:first_name, "Sarah")
     result = er.sort_queue(:state)
 
     assert_equal "20009", result.first.zipcode
```

Comment:
if this is sorting the queue by state, you could

```assert_equal "20009", result.first.zipcode```
AND
```assert_equal "<a zipcode greater than 20009>", result.last.zipcode```

to simply show that it is ordered by their zipcode.